spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+RSpec.describe "as a visitor" do
+  describe "when I click on a user's name for any book review" do
```

Comment:
hm, this doesn't seem relevant for all the tests inside it, only for the first one -- consider moving that one out & adding a different describe block for the majority of the show page tests

Reply:
Agreed, fixed in branch